.coafile

```diff
@@ -25,3 +25,7 @@ bears = HTMLLintBear
 files = **.html
 htmllint_ignore = optional_tag, protocol, indentation,  concerns_separation, capitalization
 ignore = resources/vendors/**, _site/**
+
+[yml]
+bears = YAMLLintBear
+files = **.yml
```

Comment:
Hi,

it looks like you're missing a railing newline before the end of the file here. [This article](http://stackoverflow.com/a/5813359/3212182) has more information about this issue.

Reply:
done :)